twine/repository.py

```diff
@@ -159,17 +148,26 @@ def _upload(self, package: package_file.PackageFile) -> requests.Response:
                 ("content", (package.basefilename, fp, "application/octet-stream"))
             )
             encoder = requests_toolbelt.MultipartEncoder(data_to_send)
-            with ProgressBar(
-                total=encoder.len,
-                unit="B",
-                unit_scale=True,
-                unit_divisor=1024,
-                miniters=1,
-                file=sys.stdout,
+
+            with rich.progress.Progress(
+                "[progress.percentage]{task.percentage:>3.0f}%",
+                rich.progress.BarColumn(),
+                rich.progress.DownloadColumn(),
+                "•",
+                rich.progress.TimeElapsedColumn(),
+                rich.progress.TimeRemainingColumn(),
+                "•",
+                rich.progress.TransferSpeedColumn(),
```

Comment:
This feels kind of cluttered, esp. the two time columns. Those always show `H:MM:SS`, even though `H` will almost always be "0". I'm tempted to write a custom column to shorten that, and/or drop `TimeElapsedColumn`.

Suggestions welcome.

Reply:
I guess I'm wondering if there's value in trying to match the old information given it wasn't well explained and it seems like this isn't what people might care about. Personally, if it were me, I'd rather have `TimeElapsedColumn` than `TimeRemainingColumn` if only because the former is accurate and the latter is a guess that won't always be correct and could mislead people

Reply:
> Maybe Twine doesn't need time columns at all. wink

I'm okay with that too. We can simplify and if people complain that they're not getting what they need from the animation, we can selectively add things back in.